OBAKit/WebView/DocumentWebView.swift

```diff
@@ -25,20 +25,23 @@ class DocumentWebView: WKWebView {
     /// - Parameter htmlFragment: The content to render in the web view.
     /// - Parameter actionButtonTitle: The title of the optional button shown at the bottom of the web view.
     func setPageContent(_ htmlFragment: String, actionButtonTitle: String? = nil) {
-        var content = pageBody.replacingOccurrences(of: "{{{oba_page_content}}}", with: htmlFragment)
-        content = content.replacingOccurrences(of: "{{{accent_color}}}", with: accentHexColor)
-        content = content.replacingOccurrences(of: "{{{accent_foreground_color}}}", with: accentForegroundColor)
-
-        if let actionButtonTitle {
-            let buttonText = """
-            <div class="actions__button-container">
-                <button type="button" class="actions__button-container__button" onclick="window.webkit.messageHandlers.actionButtonClicked.postMessage({})">
-                    \(actionButtonTitle)
-                </button>
-            </div>
-            """
-            content = content.replacingOccurrences(of: "{{{oba_page_actions}}}", with: buttonText)
-        }
+        var content = pageBody
+            .replacingOccurrences(of: "{{{oba_page_content}}}", with: htmlFragment)
+            .replacingOccurrences(of: "{{{accent_color}}}", with: accentHexColor)
+            .replacingOccurrences(of: "{{{accent_foreground_color}}}", with: accentForegroundColor)
+
+        // Construct button HTML if actionButtonTitle is present and not empty
+        let buttonText: String? = actionButtonTitle.flatMap { title in
+            !title.isEmpty ? """
```

Comment:
I'd prefer seeing this expressed as `title.isEmpty ? nil : """ <html content goes here> """`

* https://medium.com/@Cuadraman/why-to-stop-writting-negavite-code-af5ffb17195
* https://softwareengineering.stackexchange.com/questions/350472/developer-insists-if-statements-shouldnt-have-negated-conditions-and-should-al

Reply:
Understood. Will fix this now!